config/database.yml

```diff
@@ -3,7 +3,9 @@ development:
   adapter: postgresql
   encoding: unicode
   database: parliamentary-questions_dev
-  host: localhost
+  username: <%= ENV['DB_USERNAME'] || ENV['PGUSER'] || 'postgres' %>
```

Comment:
It's worth highlighting that this change potentially affects local development environments:

without a host parameter, rails will attempt a local unix domain socket connection. With that value present, even with the string `localhost`, rails will perform a TCP connection. Postgres naturally supports both of this, but the default access granted by a brew-installed postgres is to accept domain socket connections only.

Ways around this would be to:
1) Use docker compose (such that the pg container it brings up will have the correct access granted from the app)
2) Have a brew-installed postgres and run the app locally (i.e. not in a container), and remember to edit your local copy of this `database.yml` file to remove the `host` parameter.
3) Have a brew-installed postgres and run the app locally (i.e. not in a container), and then edit the postgres config file `pg_hba.conf` to grant host based access.